internal/cfgplugins/isis.go

```diff
@@ -0,0 +1,64 @@
+// Package cfgplugins provides configuration plugins for network protocols, including IS-IS related configuration helpers.
+package cfgplugins
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/openconfig/featureprofiles/internal/deviations"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/gnmi/oc"
+	"github.com/openconfig/ygot/ygot"
+)
+
+// ISISConfig holds all parameters needed for configuring ISIS on the DUT.
+type ISISConfig struct {
+	InstanceName string
+	AreaAddress  string
+	SystemID     string
+	AggID        string
+	Ports        []*ondatra.Port
+	LoopbackIntf string
+}
+
+// ConfigureISISOnDUT configures ISIS on the DUT using OpenConfig.
+// It enables ISIS globally, sets AFs, and applies interface-level config.
+func ConfigureISISOnDUT(t *testing.T, dut *ondatra.DUTDevice, cfg ISISConfig) *oc.NetworkInstance_Protocol {
```

Comment:
This function should return a gnmi batch object.  See https://github.com/openconfig/featureprofiles/blob/main/internal/cfgplugins/sflow.go